apps/agile-project-fe/src/app/components/list/list.stories.tsx

```diff
@@ -0,0 +1,48 @@
+/* eslint-disable */
```

Comment:
Why do we need to disable eslint here?

Reply:
We don't need it, the file was generated with this and I forgot to remove it. Sorry about that.